PmlUnit/CodeEditor.cs

```diff
@@ -83,6 +85,7 @@ enum CodeEditorKind
         Other = 0,
         Atom = 0x41544f4d, // ATOM
         NotepadPlusPlus = 0x4e502b2b, // NP++
+        PMLStudio = 0x504d4c53, // PMLS
```

Comment:
rename to `PmlStudio`
```suggestion
        PmlStudio = 0x504d4c53, // PMLS
```